apps/src/code-studio/progressRedux.js

```diff
@@ -559,6 +560,19 @@ export function statusForLevel(level, levelProgress) {
   ) {
```

Comment:
on line 549 above there is a comment talking about "assessment levels". we should come up with a consistent terminology that allows us to distinguish between those and the kind of assessments you're adding. can we call the old ones "multi-page assessment levels"? we don't need to fix all comments right now, but I think it would be helpful to update the particular comment on line 549 now since otherwise the comments within this method are a bit confusing.